script.js

```diff
@@ -5,14 +5,15 @@ const checkBtn = document.querySelector('.game-slide__check-btn');
 const helpBtn = document.querySelectorAll('.btn-help__item');
 
 const questions = document.querySelectorAll('.question');
+const questionsContainer = document.querySelector('.game-slide__questions');
 const startSlide = document.querySelector('.start-slide');
 const gameSlide = document.querySelector('.game-slide');
 const endSlide = document.querySelector('.end-slide');
 const slides = [startSlide, gameSlide, endSlide];
 
+let requestCorrectAnswers = [];
 const correctAnswers = ['Можно во всех перечисленных', 'for, while, do while',
-  'if (i == 2)', 'Сравнивает без приведения типа', 'Никакая из вышеперечисленных'
-];
+  'if (i == 2)', 'Сравнивает без приведения типа', 'Никакая из вышеперечисленных'];
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/voronkova-ab/-game-Who-want-to-be-a-millionaire-/pullRequest?prid=7740148)
```suggestion
  "if (i == 2)", 'Сравнивает без приведения типа', 'Никакая из вышеперечисленных'];
```